task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -162,16 +161,78 @@ spec:
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
+
         if ! ${OPM_BINARY} validate ."${conffolder}"; then
           echo "!FAILURE! - opm validate check failed."
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if ${OPM_BINARY} render ."${conffolder}" | jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) < 1'; then
+
+        OPM_RENDERED_CATALOG=/tmp/catalog.json
+        ${OPM_BINARY} render ."${conffolder}" > ${OPM_RENDERED_CATALOG}
+        if [ ! -f ${OPM_RENDERED_CATALOG} ]; then
+          echo "!FAILURE! - unable to render the fragment FBC."
+          failure_num=`expr $failure_num + 1`
+          TESTPASSED=false
+        fi
+
+        if jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) < 1' ${OPM_RENDERED_CATALOG}; then
           echo "!FAILURE! - There are no olm package entries defined in this FBC fragment."
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
+
+        if ! jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) == 1' /tmp/ ${OPM_RENDERED_CATALOG}; then
```

Comment:
This check was dropped as part of [KFLUXBUGS-1554](https://issues.redhat.com/browse/KFLUXBUGS-1554), so now the only check is the one above which checks if there's at least one package defined in the rendered catalog.

Reply:
My bad.  When I was resolving rebase conflicts this looked like an additional check, but I'll drop the older check in favor of this.

Reply:
Since I'd added info to the USAGE.md doc about the original check, I replaced with info about the new check instead: https://github.com/konflux-ci/build-definitions/pull/1327/files#diff-a87221e9ea400bc7daf673722fdfb1fc2a538c2d066e61e284e52a0385129e95R16-R17